mesonbuild/build.py

```diff
@@ -452,10 +452,51 @@ def get_outputs(self, backend: 'Backend') -> T.List[str]:
             for source in self.get_sources(self.srclist, self.genlist)
         ]
 
+
+@dataclass(eq=False, order=False, )
```

Comment:
Drop the trailing comma and space? Not sure what the style is here.

Reply:
it is, "there used to be another value here, but it got removed :)